azure-ts-appservice/README.md

```diff
@@ -34,7 +34,7 @@ with App Service.
 1. Define SQL Server password (make it complex enough to satisfy Azure policy):
 
     ```
-    pulumi config set azpulumi:sqlPassword <value>
+    pulumi config set <stack>:sqlPassword <value>
```

Comment:
You can leave the `<stack>:` off, and Pulumi will default to a config variable for the current stack - `pulumi config set sqlPassword <value>`

Reply:
As I said, it seems that default is project name not stack name. I got error at least.